analytics/ab_test.md

```diff
@@ -66,3 +66,11 @@ This example is with Repeat By.
 
 This example is with Repeat By. 
 ![](images/ab_prior_repeatby.png)
+
+## R Package
+
+The `A/B Test - Bayesian` uses [`bayesAB`](https://cran.r-project.org/web/packages/bayesAB/index.html) R Package under the hood.
```

Comment:
Using code quote for title of analytics looks strange to me. How about putting it this way?

"A/B Test - Bayesian" Analytics uses ...